README.md

```diff
@@ -299,27 +296,30 @@ column is configured, the connector will perform the on each run:
    at the same [MarkLogic server timestamp](https://docs.marklogic.com/guide/rest-dev/intro#id_68993) at which the initial 
    query was run. This value is then used on the next run to constrain the user's query. 
    
-A user can choose any column returned by their query. In practice, this will often be a dateTime column or a column with
-incrementing numbers, but the feature will work as long a "greater than" operation can be performed on the column. 
-Users should note the following guidelines though:
+A user can choose any column returned by their query, though the following guidelines are recommended:
 
-1. It is recommended for the column to contain unique values. For example, if the column contains numbers, an initial 
+1. The "greater than" operation should be meaningful on values in the column. In practice, the column will often be a 
+   dateTime column or a numeric column (e.g. in the case of a numeric incrementing identifier). But as long as "greater than"
+   can be used to select new or modified rows, the column should work properly.
+2. The column should ideally contain unique values. For example, if the column contains numbers, an initial 
    query may find that the highest value in the column is 50. If it is possible for future rows to be inserted with a 
    value of 50, those rows will not be returned because the connector will constrain on rows with a value greater than 50.
    Note that a "last updated" dateTime column does not need to contain unique values, assuming that any new row would 
    always have a value greater than any existing row. 
-2. A query can make use of the Optic `limit()` modifier to limit how many rows are returned in conjunction with a 
-   constraint column as well. This may be useful in situations where the number of "new" rows can be high enough to cause
-   the Kafka Connect process to run out of memory. When doing so though, the user will typically want to order the 
-   results by the same column used as the constraint column to ensure that no rows are missed. 
+3. If using the row limit feature described above, the rows should be sorted in ascending order based on the 
+   constraint column. This ensures that no rows will be skipped. 
    
-Consider the following example of using a constraint column and a limit together:
+Consider the following example of using a constraint column with a row limit:
 
-    ml.source.optic.query=op.fromView('demo', 'purchases').where(op.sqlCondition('price_per_unit > 10')).orderBy(op.asc('id')).limit(1000)
+    ml.source.optic.query=op.fromView('demo', 'purchases').where(op.sqlCondition('price_per_unit > 10')).orderBy(op.asc(op.col('id'))
```

Comment:
The user query should not include the .where or .orderBy clauses.

Reply:
This "where" clause is unrelated and can be kept - I'll remove the "orderBy" clause and make it clear that it'll be included.